Add `CONTRIBUTING.md`

There doesn't need to be a formal `CONTRIBUTING.md` file, but it would be nice to have some additional build instructions in Markdown or as script automation. `build.fsx` appears to be used for pushing to NuGet?

For example, running

```
dotnet test .\src\Fli.Tests\
```

on a clean clone yields

```
error MSB3073: The command "dotnet paket restore" exited with code 1.
```

But to run `dotnet paket restore` you need to run `dotnet tool restore` first.

Not too hard to figure out but it might improve devex for first-time contributors.

--- a/README.md
+++ b/README.md
@@ -294,5 +294,17 @@ Provided `Fli.WindowStyle`:
 ### Do you miss something?
 Open an [issue](https://github.com/CaptnCodr/Fli/issues) or start a [discussion](https://github.com/CaptnCodr/Fli/discussions).
 
+### Contributing
+After cloning this repository, there are some steps to start:
+1. `dotnet tool restore`
+2. `dotnet paket restore`
+3. `dotnet restore`
+4. `dotnet paket install`
+5. `dotnet build`
+
+After that, you can start coding, build and test.
+
+Every contribution is welcome. :)
+
 ### Inspiration
 Use CE's for CLI commands came in mind while using [FsHttp](https://github.com/fsprojects/FsHttp).

